css-scroll-snap-2/Overview.bs

```diff
@@ -70,36 +71,6 @@ First Layout {#first-layout}
 Motivating Examples {#examples}
 ===============================
 
-		<div class="example">
-				A carousel that starts in the middle:
```

Comment:
I think this is still a good example, it just requires that we add a class to the middle item in the carousel which sets scroll-start-target. Can you update the example instead?

Reply:
Sounds good, done.